spark/src/main/scala/smile/data/SparkDataFrame.scala

```diff
@@ -1,35 +1,23 @@
-/*******************************************************************************
```

Comment:
Please keep the copyright header (for all files).

Reply:
yep my bad, btw https://github.com/sbt/sbt-header could be used to make this process automatic